java/dds/OpenDDS/.gitignore

```diff
@@ -2,3 +2,4 @@
 /MAX_USER_DOMAINID.java
 /TRANSPORTTYPE_QOS_POLICY_ID.java
 /TRANSPORTTYPE_QOS_POLICY_NAME.java
+/UNALIGNED_DATA_REPRESENTATION.java
```

Comment:
This doesn't match the IDL constant.  Not sure why GitHub Actions isn't flagging it as a problem.